website/content/en/preview/getting-started/scripts/step11-create-provisioner.sh

```diff
@@ -0,0 +1,26 @@
+cat <<EOF | kubectl apply -f -
+apiVersion: karpenter.sh/v1alpha5
+kind: Provisioner
+metadata:
+  name: default
+spec:
+  requirements:
+    - key: karpenter.k8s.aws/instance-family
+      operator: In
+      values: [c, m, r]
+    - key: karpenter.k8s.aws/instance-generation
```

Comment:
we should filter to only be nitro instances too

Reply:
IMO we should just rely on Karpenter's defaults, here.

Reply:
Correct me if I'm wrong, but these seem to be the instance-family and instance-generation defaults. If that is true, do you want me to cut them, rather than show them here explicitly? Or should I just leave them as they are?